cmd/admin/cert.go

```diff
@@ -234,6 +254,12 @@ func (a *admin) revokeSerials(ctx context.Context, serials []string, reason revo
 		go func() {
 			defer wg.Done()
 			for serial := range work {
+				serial, serialErr := a.checkValidSerial(serial)
+				if serialErr != nil {
+					a.log.Errf("invalid serial format: %s", serial)
```

Comment:
This can also become more descriptive and more general at the same time, and use the `%q` format verb to ensure the input serial is quoted:
```suggestion
					a.log.Errf("skipping serial %q: %s", serial, err)
```